docs/pages/reference/config.mdx

```diff
@@ -9,6 +9,39 @@ Teleport uses the YAML file format for configuration. A full configuration refer
 file is shown below, this provides comments and all available options for `teleport.yaml`
 By default, it is stored in `/etc/teleport.yaml`.
 
+<Admonition type="note" title="Config v3">
```

Comment:
I don't think we should announce config changes at the top of the configuration reference, since this could get cluttered quickly. Instead, I think it's okay to change the configuration reference to include the new fields in `master` without including this Admonition. Users who view the v10 version of the docs will still see the previous instructions.

If migrating from `v2` to `v3` is complex, I think we should look into adding a separate, dedicated migration guide so the instructions are more prominent.

Are the old config fields still accepted in v11, i.e., while printing a warning log about deprecation etc.?

Reply:
I'd added this based on @zmb3's recommendation. I think it's nice to have, but I'm open to removing it.

I don't think the upgrade is complex, this is quite a short note on what's changed that I don't think warrants another page.

If you're using config v3, the old config values are not accepted and there will be an error. However, we always assume config v1 if a version isn't specified, and if a config already has a version set it'll be v1 or v2. This means that upgrading to config v3 is a conscious effort, so we don't need to add warnings about deprecation.

Reply:
This has been removed